lib/modules/newCommentCount.js

```diff
@@ -286,16 +286,19 @@ function subscribe(threadid) {
 	drawSubscriptionsTable();
 }
 
-function displayNewCommentCount(thing) {
+export function getNewCount(thing: Thing): ?number {
```

Comment:
For this purpose, it might be better to surface `export function isPostVisited(thing: Thing): bool` to handle the case where the user has visited the post but there aren't new comments since then.